makefile

```diff
@@ -1,44 +1,44 @@
 ifneq ($(OS_NAME),)
-	TARGET_OS = $(OS_NAME)
+    TARGET_OS = $(OS_NAME)
 else
-	ifeq ($(OS),Windows_NT)
-		TARGET_OS = Windows
-	else ifeq ($(PLATFORM),iPhoneOS)
-		TARGET_OS = iPhoneOS
-	else ifeq ($(PLATFORM),iPhoneSimulator)
-		TARGET_OS = iPhoneSimulator
-	else
-		UNAME_S := $(shell uname -s)
-		ifeq ($(UNAME_S),Darwin)
-			TARGET_OS = Mac
-		else
-			TARGET_OS = Linux
-		endif
-	endif
+    ifeq ($(OS),Windows_NT)
+        TARGET_OS = Windows
+    else ifeq ($(PLATFORM),iPhoneOS)
+        TARGET_OS = iPhoneOS
+    else ifeq ($(PLATFORM),iPhoneSimulator)
+        TARGET_OS = iPhoneSimulator
+    else
+        UNAME_S := $(shell uname -s)
+        ifeq ($(UNAME_S),Darwin)
+            TARGET_OS = Mac
+        else
+            TARGET_OS = Linux
+        endif
+    endif
 endif
 
 ifeq ($(TARGET_OS),Windows)
-	LIBRARY = binn-3.0.dll
+    LIBRARY = binn-3.0.dll
 else ifeq ($(TARGET_OS),iPhoneOS)
-	LIBRARY = libbinn.dylib
-	CFLAGS += -fPIC
+    LIBRARY = libbinn.dylib
+    CFLAGS += -fPIC
 else ifeq ($(TARGET_OS),iPhoneSimulator)
-	LIBRARY = libbinn.dylib
-	CFLAGS += -fPIC
+    LIBRARY = libbinn.dylib
+    CFLAGS += -fPIC
 else
-	ifeq ($(TARGET_OS),Mac)
-		LIBRARY = libbinn.3.dylib
-		LINK1   = libbinn.dylib
-	else
-		LIBRARY = libbinn.so.3.0
-		LINK1   = libbinn.so.3
-		LINK2   = libbinn.so
-	endif
-	CFLAGS += -fPIC
+    ifeq ($(TARGET_OS),Mac)
+        LIBRARY = libbinn.3.dylib
+        LINK1   = libbinn.dylib
+    else
+        LIBRARY = libbinn.so.3.0
+        LINK1   = libbinn.so.3
+        LINK2   = libbinn.so
+    endif
+    CFLAGS += -fPIC
```

Comment:
almost done... now just please replace the spaces back to tab and we are done

Reply:
To keep some sort of indentation, I converted them to spaces, which are ignored by make.
Tabs, as they were, for a non rule, are invalid.

Please read:

> Extra spaces are allowed and ignored at the beginning of the conditional directive line, but a tab is not allowed. (If the line begins with a tab, it will be considered part of a recipe for a rule.) Aside from this, extra spaces or tabs may be inserted with no effect anywhere except within the directive name or within an argument. A comment starting with ‘#’ may appear at the end of the line.

More on this: https://www.gnu.org/software/make/manual/make.html#Conditional-Syntax